src/start-stop-daemon/start-stop-daemon.c

```diff
@@ -191,6 +194,22 @@ static inline int ioprio_set(int which RC_UNUSED,
 }
 #endif
 
+#if !defined(SYS_close_range) && defined(__NR_close_range)
```

Comment:
I really don't think we should do this. Using `close_range` is fine, but needing hacks for it isn't.

musl should provide a prototype for it?

Reply:
I think your current approach is alright, even if the situation isn't ideal.

It would be nice if the musl maintainers had replied when you asked about it though :(

Consider my objection withdrawn, thank you for working on this!

Reply:
FWIW, I'm doing a second try of including close_range() in musl: https://www.openwall.com/lists/musl/2023/09/01/1